runtime/Cargo.toml

```
@@ -18,15 +21,16 @@ threads = 8
 debug = false
 
 [dependencies]
-keymanager = { git = "https://github.com/oasisprotocol/keymanager-paratime", tag = "v0.4.1-testnet" }
+pontusx-keymanager = { git = "https://github.com/oasisprotocol/pontusx-keymanager", tag = "v0.0.1-testnet" }
```

I would change it to the following so that we don't need to update the key manager package in code.
```suggestion
keymanager = { package = "pontusx-keymanager", git = "https://github.com/oasisprotocol/pontusx-keymanager", tag = "v0.0.1-testnet" }
```